docs/docs/docs/01-core/database/02-transactions.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Transactions
+description: Conceptual guide to transactions in Strapi
+tags:
+  - database
+---
+
+This provides an API to wrap a set of operations in a transaction that ensures the integrity of data.
```

Comment:
what is "this"? :) would it be better to say "transactions are an API to wrap..."?

Reply:
Isn't a bit redundant to say, transactions are an API to wrap transactions?

I would like to say "this" api is used to create transactions.

Would it be better to say, 
`API to wrap a set of poperations in a transaction...`

Reply:
"An API to wrap a set of operations that ensures the integrity of data."?

Reply:
sounds good 👍 changing it!

Reply:
reworded to `An API to wrap a set of operations in a transaction that ensures the integrity of data.`